### PR TITLE
Careers

### LINKED ISSUE #4: Add Careers

### PR DESCRIPTION
**Pull request by janklimo**
>https://careers.drrxiultpxaar.amplifyapp.com/
> **Username**: vmx
> **Password**: xeLQqk6kVrn.7wz

Screen Shot 2019-11-19 at 5 13 33 PM

Resolves #4

### CODE REVIEW

**Comment by janklimo** [comment]
@hiattp I'm thinking the best way to collect applications will be to have an online form (like Google Forms)  so we keep the data organized. Do you agree? Does Office have something similar? A careers mailbox is still useful - we can send the applicant a copy of their submitted application from it.

**Comment by hiattp** [comment]
@janklimo let's follow a "progressive enhancement" strategy here and just stick with an email for now (tech.recruiting@verumex.com, I set that up a few weeks ago and should work for this). Historically the inbound volume is low but if we get inundated enough that we feel unorganized we can definitely set up a form. My only issue with that is it's just One More Thing to keep track of... would rather not go down that route if we don't need to yet 😁

**Comment by hiattp** [comment]
Didn't go with careers@verumex.com just because it is a bit too general (we'll be recruiting for lots of different positions so a single generic email will get complicated). But I can change that to tech.careers@verumex.com if you think that has a better ring to it?

**Comment by janklimo** [comment]
Hey @Verumex/core can I get some 👀 on this?

I've created 2 job posts: junior (growth focus), and senior (experience focus).

Things to discuss before we hit publish:

1. Offer. There are some extra points under _Ideas_ page (not to be published) that I've come across in my research that we don't do yet. Let's discuss which ones are possible/attractive. Did I forget anything?
2. Application. Looking for a balance between a process that isn't too demanding and something good enough at filtering out less-than-very-interested applicants.
3. I felt like the posts would ideally go deeper into what we do and what the person would actually do day to day (useful when we repost to other media) but I didn't really know what to write + it's already getting quite long. I think a lot of this is self-evident and covered on other pages which I've linked.

File: content/careers/senior-rails-engineer.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+You'll be responsible for scaling, monitoring, and optimizing system performance.
+
+Your contributions will be continuously deployed and used by the world's
+largest real estate funds.
+
+You'll be part of a high-functioning team that is here to help wherever
+we can. Our company is growing fast and so will you.
+
+<Offer salary="200,000" />
```
Comment (janklimo) [inline content/careers/senior-rails-engineer.mdx]: MDX = 🤯

File: content/careers/senior-rails-engineer.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## How Do I Apply?
+
+Please send an email to
+[tech.recruiting@verumex.com](mailto:tech.recruiting@verumex.com?subject=Senior%20Rails%20Engineer)
+with the following information:
+
+1. Your **CV or LinkedIn**.
+2. Link to your **Github** profile.
```
Comment (karunsiri) [inline content/careers/senior-rails-engineer.mdx]: I personally think we should welcome any source control service profiles. I also have a lot of Bitbucket projects that doesn't overlap with Github project. Some people might worked at corporate companies that host their own Gitlab or source control services. Apparently the profile must be accessible (public) 😁

**Comment by hiattp** [comment]
I think I addressed those concerns in deb83d3 (making all of those requirements optional). That should get us detailed responses from interested people while not scaring anyone off if they for some reason feels like their CVs are sufficient. Looking good!

Only other comments:
1. We need to restructure this slightly to allow for country-specific careers pages/listings. I'll want to put Chennai versions of these up as soon as this is merged 😄.
2. Can you guys each rank the items from the Ideas page (maybe in a shared excel doc or something) so we can internally get a sense of how interesting/valuable those other perks are to everyone? Everything is on the table there, but let's prioritize a few and get them in motion (as well as include them in the job descriptions).

File: content/careers/senior-rails-engineer.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+3. Explain why you think you're a great **cultural fit**. We're firm believers
+   in hiring people who [share our values and engineering philosophy](/team).
+4. **Tell us about your programming experience**.
+   [Our platform](/platform) is built with Ruby on Rails, PostgreSQL and
+   RSpec. It would be great if you have experience with this stack but even
+   if you don't, explain what other technologies you've used and show us
+   any **examples** of what you've built.
```
Comment (janklimo) [inline content/careers/senior-rails-engineer.mdx]: @hiattp it's ok to be flexible for juniors but I think experience with this basic stack should be a hard requirement for seniors. What do you think?

**Comment by janklimo** [comment]
@hiattp I added another tree level to support different hiring locations. Chennai is a placeholder you can override with your content.

**Comment by janklimo** [comment]
@Verumex/core can you please add your answers to [this spreadsheet](https://docs.google.com/spreadsheets/d/1A5aJazFpO2izqvUu3ULy_v87Nr2bvITXQuDtOLPDpwc/edit?usp=sharing)

File: content/careers/senior-rails-engineer.mdx (hunk 3 above)
Comment (hiattp) [reply content/careers/senior-rails-engineer.mdx]: I actually think it depends—if someone has a ton of experience with a non-rails grape, minitest and mysql stack I suspect they would pick up our stack fast enough. If they have zero experience with ruby I doubt they would make it through the process but ultimately until we have a problem of too many applications I don't think we need to lock down those filter criteria yet. Easy enough to change later

**Comment by hiattp** [comment]
Looks good let's get it out and start fielding some resumes, we can continue to tweak as we go

**Comment by hiattp** [comment]
@janklimo assigning to you in case you want to squash my latest commits (up to you)

**Comment by janklimo** [comment]
@hiattp does Amplify have Github integration? Netlify has a nice one with a link to preview app (and [lots of other things](https://docs.netlify.com/site-deploys/notifications/#github-commit-checks)):

preview

I suppose we went with Amplify to keep all services under AWS?

**Comment by hiattp** [comment]
Yep amplify has the same sort of gh integration I just didn’t turn it on, it seems comparable to netlify wrt the features we might actually use and does have that added bonus of living in our aws umbrella

**Pull request merged**